test/cljs/rems/administration/test_create_catalogue_item.cljs

```diff
@@ -46,3 +46,107 @@
     (testing "missing form"
       (is (nil? (build-request (assoc form :form nil)
                                languages))))))
+
+(deftest resource-label-test
+  (let [resources [{:resid "x"
+                    :licenses [{:localizations {:en {:title "Performance License"}}}
+                               {:localizations {:en {:title "Second License"}}}]
+                    :organization {:organization/id "nbn"
+                                   :organization/short-name
+                                   {:en "NBN en"
+                                    :fi "NBN fi"
+                                    :sv "NBN sv"}}
+                    :owneruserid "owner"}
+                   {:resid "y"
+                    :licenses [{:localizations {:en {:title "Performance License"}
+                                                :fi {:title "Performance License FI"}
+                                                :sv {:title "Performance License SV"}}}
+                               {:localizations {:en {:title "Second License"}
+                                                :fi {:title "Second License FI"}
+                                                :sv {:title "Second License SV"}}}]
+                    :organization {:organization/id "nbn"
+                                   :organization/short-name {:en "NBN en"
+                                                             :fi "NBN fi"
+                                                             :sv "NBN sv"}}}
+                   {:resid "y"
+                    :licenses [{:localizations {:en {:title "Performance License"}
+                                                :fi {:title "Performance License FI"}
+                                                :sv {:title "Performance License SV"}}}]}
+                   {:resid "u"}
+                   {:resid "z"}]
+        counts (frequencies (map :resid resources))]
+    (testing "resource-label"
+      (is (= ["y ([:t.administration/org]: NBN en) ([:t.administration/licenses]: Performance License, Second License)"
+              "y ([:t.administration/licenses]: Performance License)"]
+             (map #(resource-label % :en counts) [{:resid "y"
```

Comment:
And the `map` is still here?

Reply:
I did not delete it, I added more tests cases.

Reply:
You marked the discussion as resolved?

Reply:
this one? no, obviously

Reply:
I can delete the test altogether